requirements-dev.in

```diff
@@ -1,6 +1,7 @@
 -r requirements.txt
 -e .
 
-honcho
 ipython
 ipdb
+supervisor
+docker-compose
```

Comment:
To allow supervisor to run docker-compose it was necessary to remove the separate docker-compose tox env and install docker-compose in the dev env that supervisor is run in. This makes sense: NGINX is a dev process not a service (in other apps the separate docker-compose env is used for running services)